schemas/dependencies/dynatrace-environment-1.yml

```diff
@@ -14,9 +14,27 @@ properties:
   description:
     type: string
   environmentUrl:
+    description: the URL to a specific Dynatrace entity called environment, which is also known as tenant.
     type: string
   bootstrapToken:
     "$ref": "/common-1.json#/definitions/vaultSecret"
+  environment:
```

Comment:
Not super happy with the name, since we already use `environmentUrl` above which is sth completely different. Maybe we can add a description here that this purely targets different stages?